lib/worker-pipeline.js

```diff
@@ -69,6 +70,29 @@ class AssetComputeWorkerPipeline {
         return engine;
     }
 
+    normalizeInputOuput(input, output) {
+        console.log('input before normalization: ', input);
+        console.log('outpu before normalization: ', output);
+
+        if(output.fmt === 'machine-metadata-json') {
+            output.type = 'machine-metadata-json';
```

Comment:
note: we will have special cases for beta-worker-creative too

Reply:
thanks, will add a TODO